CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.0.0](https://github.com/kakawait/cas-security-spring-boot-starter/milestones) (miliestone number of v2.0.0 to be determined by owner) - 04 jul 2022
+
+- Apply pom code convention (https://maven.apache.org/developers/conventions/code.html#pom-code-convention)
+- Use java 11
+- Use spring-boot 2.6.9. (When 2.7.0 or higher will be used, it will become a breaking change thus the upgrade to a major version)
```

Comment:
Just to be sure why did you limit to 2.6.9 instead of 2.7.0 if the final target is 2.7.0?

Reply:
This PR is in preparation of solving #161

It will take some effort to go to spring-boot 2.7.0 as this will  also cause a breaking change since WebSecurityConfigurerAdapter  is deprecated in this version.

So I prefer to do this in a separte PR later